server/src/main/java/org/folio/reservoir/server/OaiService.java

```diff
@@ -384,4 +388,89 @@ static Future<Void> getRecord(RoutingContext ctx) {
               }));
     });
   }
+
+  static Future<Void> sruGetResponse(RoutingContext ctx, String query) {
```

Comment:
Should this be moved to a separate service class?

Reply:
I started with that, but it would basically rely on the oai service or the oai service would have to be split into two..

Reply:
Done